src/jit/IntMath32Inl.h

```diff
@@ -16,6 +16,11 @@
 
 /* Only included by jit-backend.cc */
 
+#if defined(COMPILER_MSVC)
+#include <intrin.h>
+#define __builtin_popcount __popcnt
```

Comment:
Maybe we could use `popCount` in `util/BitOperation.h`. This should improve future maintainability.
The header could be included in `Backend.cpp` so it would be available everywhere.